landing-page/style.css

```diff
@@ -0,0 +1,149 @@
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@300&family=Bitter:wght@600&family=Dosis:wght@200;300;600&family=Outfit:wght@200&family=Proza+Libre&family=Redressed&family=Roboto:ital,wght@1,300&family=Shippori+Antique&family=Special+Elite&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@300&family=Bitter:wght@600&family=Dosis:wght@200;300;600&family=Outfit:wght@200&family=Proza+Libre&family=Redressed&family=Roboto:ital,wght@1,300&family=Shippori+Antique&family=Special+Elite&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@300&family=Bitter:wght@600&family=Dosis:wght@200;300;600&family=Outfit:wght@200&family=Proza+Libre&family=Redressed&family=Roboto:ital,wght@1,300&family=Shippori+Antique&family=Special+Elite&display=swap');
+@import "/resetcss.css";
+@import "/variablecss.css";
+
+
+/* -----------------------------------------------------------------------------RESPONSIVE */
+@media only screen and (max-width: 768px) {
+    /* For mobile phones: */
+    html{
+        font-size: 40%;
+    }
+    [class*="col-"] {
+        width: 100%;
+       }
+       .sliding-banner img{
+        width: 100%;
+        height: auto;   
+       }
+}
+  
+  /* ------------------------------------------------------------------------------SLIDER--*/
+  .sliding-banner img{
+   width: 100%;
+   height: 60rem;   
+  }
+  .sliding-banner{
+      position: relative;
+  }
+  .fades {
+    -webkit-animation-name: fade;
+    -webkit-animation-duration: 1.5s;
+    animation-name: fade;
+    animation-duration: 1.5s;
+  }
+  
+  @-webkit-keyframes fade {
+    from {opacity: .4} 
+    to {opacity: 1}
+  }
+  
+  @keyframes fade {
+    from {opacity: .4} 
+    to {opacity: 1}
+  }
+  /* ----------------------------------dots */
+
+  .dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+    transition: background-color 0.6s ease;
+    margin-bottom: 8rem;
+  }
+  .active, .dot:hover {
+    background-color: #717171;
+  }
```

Comment:
for background colour declare a variable for this also.

Reply:
sure